### ISSUE TITLE
A question in regards to VRAM

### ISSUE THREAD

**Issue by nikmes** [libcg/gLib2D#4]
Draw & display buffers can't actually be used as real textures. Just a way   to get the vram pointer. 
I assume this library dose not utilize the VRAM. Is that assumption correct?
I do not see any valloc or vfree calls.

**Comment by libcg** [comment]
Yes. I once managed to use the framebuffer as a texture with pspgu to do a blur effect. That was the point of these... But for some reason I could never get it to work again. Otherwise all the textures are stored in RAM.

**Comment by nikmes** [comment]
Hi, I understand how it works :) I used the vmalloc statment to get memmory from VRAM instead but and i notice that only 0.6Mb remains free in VRAM after you allocate for DISPLAY, DRAW and DEPTH Buffers which is very little :) I am wondering if it worth to develop a functionality that will swap in and out textures from PSP RAM to VIDEO RAM.
